projects/2020_11_14-pkt-community.md

```diff
@@ -134,6 +134,22 @@ in this project.
 ### November 26, 2020
 Project accepted and initial payout of 12mn PKT made in transactions:
 
+### April 9, 2021
+Milestone #1:
+* VIDEO - 1 of 5 completed: https://youtu.be/_EeWHeY-omQ
+* TECH_ARTICLE - 2 of 6:
+** https://www.cryptopolitan.com/how-the-network-steward-ensures-funding-for-the-build-out-of-decentralized-internet-infrastructure-and-open-source-technology/
+** https://www.cryptopolitan.com/pkt-network-packet-priority-creates-a-more-economical-infrastructure-that-will-help-maximise-data-transmission/
+* ARTICLE - 2 of 6 completed:
+** https://docs.google.com/document/d/14G4sbUgrpuNjd9k0KrO0l10-9hnyVJLk/edit
+** https://securityonline.info/all-you-need-to-know-about-pkt-cryptocurrency/
+* GRAPHICS - 8 of 28 completed: https://drive.google.com/drive/folders/1gI_MiqkxkJZSsLobtbIddKWsB-izvA9d?usp=sharing
+* WWW_DOCS - launched with basic documentation written: https://docs.pkt.cash/en/latest/
+* WWW_BLOG - launched: http://crypto.pkt.cash/
```

Comment:
404?

Reply:
Good call, I just changed my review until this is fixed.